CMakeLists.txt

```diff
@@ -339,8 +342,21 @@ if(EMSCRIPTEN)
     add_compile_flag("-sDISABLE_EXCEPTION_CATCHING=0")
     add_link_flag("-sDISABLE_EXCEPTION_CATCHING=0")
   endif()
-  # make the tools immediately usable on Node.js
-  add_link_flag("-sNODERAWFS")
+  # make the tools immediately usable on Node.js, unless building for the browser
+  if (NOT BUILD_FOR_BROWSER)
+    add_link_flag("-sNODERAWFS")
+  endif()
+  if(BUILD_FOR_BROWSER)
```

Comment:
nit: Can you write this as "if" + "else" instead of two ifs?

Reply:
Addressed in https://github.com/WebAssembly/binaryen/pull/6131/commits/018f0aa45b1bc6a29d07bb3fc4c3d651880b36e6.